Sources/Logging/Logging.swift

```diff
@@ -21,6 +21,9 @@ public protocol LogHandler {
 
     // The log level
     var logLevel: Logger.Level { get set }
+
+    // A user-suppliable identifier
+    var label: String? { get }
```

Comment:
sorry, missed that before, this shouldn't be optional as every LogHandler gets handed a label